src/System.Memory/src/System.Memory.csproj

```diff
@@ -0,0 +1,77 @@
+    <Compile Include="$(CommonPath)\CoreLib\System\Buffers\MemoryManager.cs" Link="Common\CoreLib\System\Buffers\MemoryManager.cs" />
+    <Compile Include="$(CommonPath)\CoreLib\System\Memory.cs" Link="Common\CoreLib\System\Memory.cs" />
+    <Compile Include="$(CommonPath)\CoreLib\System\MemoryDebugView.cs" Link="Common\CoreLib\System\MemoryDebugView.cs" />
+    <Compile Include="$(CommonPath)\CoreLib\System\MemoryExtensions.cs" Link="Common\CoreLib\System\MemoryExtensions.cs" />
+    <Compile Include="$(CommonPath)\CoreLib\System\ReadOnlyMemory.cs" Link="Common\CoreLib\System\ReadOnlyMemory.cs" />
+    <Compile Include="$(CommonPath)\CoreLib\System\ReadOnlySpan.cs" Link="Common\CoreLib\System\ReadOnlySpan.cs" />
+    <Compile Include="$(CommonPath)\CoreLib\System\Runtime\InteropServices\MemoryMarshal.cs" Link="Common\CoreLib\System\Runtime\InteropServices\MemoryMarshal.cs" />
+    <Compile Include="$(CommonPath)\CoreLib\System\Span.cs" Link="Common\CoreLib\System\Span.cs" />
+    <Compile Include="$(CommonPath)\CoreLib\System\SpanDebugView.cs" Link="Common\CoreLib\System\SpanDebugView.cs" />
+    <Compile Include="$(CommonPath)\CoreLib\System\SpanHelpers.BinarySearch.cs" Link="Common\CoreLib\System\SpanHelpers.BinarySearch.cs" />
+    <Compile Include="$(CommonPath)\CoreLib\System\SpanHelpers.Byte.cs" Link="Common\CoreLib\System\SpanHelpers.Byte.cs" />
+    <Compile Include="$(CommonPath)\CoreLib\System\SpanHelpers.Char.cs" Link="Common\CoreLib\System\SpanHelpers.Char.cs" />
+    <Compile Include="$(CommonPath)\CoreLib\System\SpanHelpers.T.cs" Link="Common\CoreLib\System\SpanHelpers.T.cs" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(IsPartialFacadeAssembly)' != 'true' and $([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) != '.NETFramework'">
+    <Compile Include="System\Buffers\ReadOnlySequenceSegment.cs" />
+    <Compile Include="System\Buffers\StandardFormat.cs" />
+    <Compile Include="System\MemoryExtensions.Portable.cs" />
+    <Compile Include="System\NUint.cs" />
+    <Compile Include="System\Pinnable.cs" />
+    <Compile Include="System\ReadOnlySpan.Portable.cs" />
+    <Compile Include="System\Runtime\InteropServices\MemoryMarshal.Portable.cs" />
+    <Compile Include="System\Span.Portable.cs" />
+    <Compile Include="System\SpanHelpers.cs" />
+    <Compile Include="System\ThrowHelper.cs" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\System.Buffers\src\System.Buffers.csproj" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(IsPartialFacadeAssembly)' != 'true'">
+    <PackageReference Include="System.Numerics.Vectors" />
+    <PackageReference Include="System.Runtime.CompilerServices.Unsafe" />
+  </ItemGroup>
+
+
+  <ItemGroup Condition="$([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) == '.NETFramework'">
+    <Reference Include="System" />
+    <Reference Include="mscorlib" />
```

Comment:
Didn't know they get referenced by default in netfx. I removed them and the build succeeds.

Why were they explicit in corefx 2.1?

Reply:
Because back then, the infrastructure required explicit references for everything. I changed that between 3.1 and 5.0 to let a project use the implicit framework references (default behavior).